cas/incident-report.md

```diff
@@ -1,169 +1,387 @@
+### What format should be used?
+Use one of the templates below, depending on the type of Incident Report being disclosed:
 
-Each incident should result in a report written as soon as the problem is fully diagnosed and (temporary or permanent) measures have been put in place to ensure it will not reoccur. If the permanent fix will take significant time to implement, you should not wait until this is done before issuing the report.
+- [Preliminary Incident Report](#preliminary-incident-report) (for third party reporters and CA Owners)
+- [Full Incident Report](#full-incident-report) (for CA Owners)
 
-An initial report should be filed within 72 hours of the CA Owner being made aware of the incident. If a full incident report is not yet ready, CA Owners should provide a preliminary report containing an executive summary of the incident and a date by which the full report will be posted. The full incident report must be posted within two weeks of the incident. Once the report is posted, CA Owners should respond promptly to questions that are asked, and in no circumstances should a question linger without a response for more than one week, even if the response is only to acknowledge the question and provide a later date when an answer will be delivered. 
+If being reported by the CA Owner corresponding with an incident, all fields included in the relevant template MUST be completed.
 
-There should be a single incident report for each distinct matter, and CA Owners should submit an additional, separate incident report when:
+### When are Incident Reports expected?
 
-- Policy requires the revocation of one or more certificates by a certain deadline, such as those in BR section 4.9, but that deadline will not be or has not been met by the CA Owner.
-- In the process of researching one incident, another incident with a distinct root cause and/or remediation is discovered.
-- After an incident is marked resolved in Bugzilla, the incident reoccurs.
+Within 72 hours of the CA Owner being made aware of an incident (i.e., "initial incident disclosure"), the CA Owner MUST 
 
-The incident report may well repeat things previously said in discussions or Bugzilla comments. This is entirely expected. The report should be a summary of previous findings. The existence of data in discussions or Bugzilla comments does not excuse a CA Owner from the task of compiling a proper incident report.
+Either:
+1) disclose a Preliminary or Full Incident Report; or
+2) respond to a Preliminary Incident Report previously created for the incident by a third party reporter.
 
-Open incident reports should be updated:
+And:
+- accurately disclose the impact of the incident (e.g., a corpus of mis-issued certificates).
```

Comment:
Maybe this should be re-phrased to (e.g., the corpus of then-known mis-issued certificates)?  Also, I am working on cross-referencing this language instead of repeating it in the MRSP.  See https://github.com/mozilla/pkipolicy/issues/270.

Reply:
#187 rephrases this, as suggested by Ben above.